src/__tests__/lmdb-cache-prune.test.ts

```diff
@@ -0,0 +1,71 @@
+import { LmdbCache } from '../LmdbCache';
+import { defaultCacheOptions, PruneStats } from 'warp-contracts';
+import * as fs from 'fs';
+
+describe('Lmdb cache prune', () => {
+  beforeEach(() => {
+    fs.rmSync('./cache', { force: true, recursive: true });
```

Comment:
are you sure that it will not fail if the ./cache dir is not created?

Reply:
That's what `force` is for. On Unix if you ad force removing it skips errors